heareval/tasks/speech_commands.py

```diff
@@ -2,40 +2,46 @@
 """
 Pre-processing pipeline for Google Speech Commands
 """
+# from heareval.tasks.coughvid import DownloadCorpus, ExtractCorpus
 import os
+import subprocess
 
 import luigi
 import numpy as np
 
 import heareval.tasks.config.speech_commands as config
 import heareval.tasks.util.audio as audio_util
 import heareval.tasks.util.luigi as luigi_util
-from heareval.tasks.util.luigi import WorkTask
+from heareval.tasks.util.luigi import DownloadCorpus, WorkTask, ExtractZipFile
 
 
 # Set the task name for all WorkTasks
 WorkTask.task_name = config.TASKNAME
 
 
+class DownloadExtractCorpus(ExtractZipFile):
```

Comment:
This adds the requirement of download the extract zip file. Was confused about how to get the output location from the download to the extract. Though of this as a way. Please recommend if anything else is suitable.

Reply:
As discussed issue with pickling lambda function. Possible ways is the list constructor options or passing the task as a `prev_task = luigi.TaskParameter()` and using it in requires like `self.prev_task`